plugins/content/course/index.js

```diff
@@ -47,6 +47,7 @@ function initialize () {
       var options = _.keys(req.body).length
       ? req.body
       : req.query;
+      var options = JSON.stringify(options);
```

Comment:
you could remove the var. Already declared in line above var options = _.keys(req.body).length